src/components/Counter/Counter.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import cl from "./Counter.module.css";
+
+interface ICounterProps {
+  initValue: number;
+}
+
```

Comment:
Better to move all the interfaces to another file

Reply:
Moved